### PR TITLE
Switched Kenya North devices

### PR DESCRIPTION
**Pull request by aforbis-stokes**
Old North device (ADPLKenyaN3763) had damaged analog input pins. New device (ADPLKenyaN6833) installed on 1/18/17. Routing web-server to new device.

### CODE REVIEW

File: software/cloud-dashboard/node-server/config/device-map.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
 module.exports={
 	locMap : {
 	"4b0031000d51353432393339": "ADPLKenyaC6304",
-        "300057000951343334363138": "ADPLKenyaN3763",
+        "3e001d000e51353432393339": "ADPLKenyaN6833",
```
Comment (suyashkumar) [inline software/cloud-dashboard/node-server/config/device-map.js]: This device is the same site as the 3763, correct? The actual mapping to name can be arbitrary, and does not need to reflect the particle's actual "name"

If we keep it `ADPLKenyaN3763` or just change it to something more generic like `ADPLKenya_1` we can keep the database data grouped together for that site across different particles and still identify the actual particle device based on the coreid :).

What're your thoughts @aforbis-stokes?

Comment (suyashkumar) [reply software/cloud-dashboard/node-server/config/device-map.js]: Does that make sense?

**Review by suyashkumar — APPROVED** [review]
Looks good to me, just check out my comments!

File: software/cloud-dashboard/node-server/config/device-map.js (hunk 1 above)
Comment (aforbis-stokes) [reply software/cloud-dashboard/node-server/config/device-map.js]: That does make sense. We could then switch simply to "Kenya - North" for that device and use the same on others (Kenya - Central, India, Philippines - Sagay, Philippines - Subayon) or something similar.

Comment (suyashkumar) [reply software/cloud-dashboard/node-server/config/device-map.js]: Yeah, that sounds good! I can run a backfill on the database to update all the old records at some point too hopefully this week.

Comment (suyashkumar) [reply software/cloud-dashboard/node-server/config/device-map.js]: I can go ahead and make the change in a separate diff or you can just make it in this one and then I'll merge to master.

If we just want to get this out--if you just keep the name `"ADPLKenyaN3763"` I can merge this to master and deploy it so we don't miss data :)

Comment (aforbis-stokes) [reply software/cloud-dashboard/node-server/config/device-map.js]: Ok, let's just keep the name "ADPLKenyaN3763" in order to get the data. We can change later.

Comment (suyashkumar) [reply software/cloud-dashboard/node-server/config/device-map.js]: Sounds good, made the change and merging now

**Pull request merged**